### ISSUE TITLE
WebToc.xml not generated using VS2013 presentation style

### ISSUE THREAD

**Issue by Venca-M** [EWSoftware/SHFB#987]
Hi,

I am following issue #981. WebToc.xml is not generated and it previously was. We went through a migration to .net6 so I suspect a problem there. The setup is pretty much as it was. In the build log there is only one mention of the WebToc.xml file and I am not sure if that is ok. It looks like file is generated, but not copied to the final output?
```
Saving website table of contents to C:\Users\Venca\Source\Repos\TSDrepo\TSD.API.Remoting.Documentation\GeneDocHelp\Working\\WebTOC.xml
    Last step completed in 00:00:00.3860
```
Here is the full log so you can inspect:
[LastBuild.log](https://github.com/EWSoftware/SHFB/files/11259122/LastBuild.log)

Thanks for your help
Venca

**Comment by EWSoftware** [comment]
It is generated but it's in the Working folder now rather than the output folder since it's only used by the Lightweight TOC plug-in to generate it's TOC files.  It isn't used by the live site at all.  Is it something you need in your live site?  If not, and it's just for another plug-in or something, you can point it to the copy in the working folder instead.

**Comment by Venca-M** [comment]
Hi @EWSoftware, 
sorry I went silent. I am getting back to trying to solve this problem. 

We need the WebTOC.xml file in the final output for some downstream work. I was able to manually snatch it from the build process, but taht is no permanent solution. 
It would be nice if I could turn off the auto delete of the Working directory after build. I think that would be enough to solve my problems.

Thanks
Venca
